crowd_anki/representation/deck.py

```diff
@@ -82,6 +84,12 @@ def _load_deck_config(self):
         new_config = DeckConfig.from_collection(self.collection, self.anki_dict["conf"])
         self.deck_config_uuid = new_config.get_uuid()
 
+        # TODO Remove this once enough time has passed that #106/#116
```

Comment:
create a separate issue to track this?

Reply:
also - does this actually help? would we write an updated version of the dict to db?

Reply:
> create a separate issue to track this and mention it in the comment?

Makes sense! Will do!

> also - does this actually help?

Yeah, it helps!

1. It ensures that the correct `deck_config_uuid` is set during that given export.

> would we write an updated version of the dict to db?

2. Yes, we do write an updated version of the deck dict to the db ([here](https://github.com/Stvad/CrowdAnki/blob/master/crowd_anki/export/anki_exporter.py#L71)), so we're safe going forward (even if we remove this code).

To double check *2*: 

(steps 1-4 are from adapted from [above](https://github.com/Stvad/CrowdAnki/pull/127#issue-624377195), but without the changing of the deck config, since we're only concerned with whether `deck_config_uuid` is present at all)

0. Use the current, published version of the add-on (or `master` from `git`).
1. Start with a fresh profile.
2. Create a deck (say `foo`) with a couple of notes.
3. Export `foo` with CrowdAnki.
4. Re-import `foo` with CrowdAnki.
5. Using the Anki console (accessed with <kbd>Ctrl</kbd><kbd>Shift</kbd><kbd>;</kbd>) check that `deck_config_uuid` is (undesirably) present in the Anki dict:

```python3
print('deck_config_uuid' in mw.col.decks.byName('foo'))
```
> `True`

6. Quit Anki, switch to the version of the add-on from this branch/PR and start Anki again.
7. (Optionally, do 5 again, getting `True`, again, to make sure that it's step 8 that removes `deck_config_uuid`, not restarting Anki etc.)
8. Export `foo` with CrowdAnki, again.
9. Like in 5 (and 7), check whether `deck_config_uuid` is present in the Anki dict for `foo`:
```python3
print('deck_config_uuid' in mw.col.decks.byName('foo'))
```
> False